<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MessageList" name="messageWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="separateLine">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="chatStackedWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MessageList</class>
   <extends>QWidget</extends>
   <header>messagewidget/messagelist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IMClientChatMain/imclientchatmain.qrc"/>
 </resources>
 <connections/>
</ui>
